Update document for `istioctl auth tls-check` command (#3343)
istio/istio.io: content/docs/tasks/security/mutual-tls/index.md

## Verify mutual TLS configuration

You can use the `istioctl` tool to check the effective mutual TLS settings. The command requires the client pod as the destination rule depends on the client namespace.
You can also provide the destination service to filter the status to that service only.

> To identify the authentication policy for the `httpbin.default.svc.cluster.local` service and its destination rules as seen from the some a pod of the `sleep` app, use the following command:

Reviewer: ```suggestion
To identify the authentication policy for the `httpbin.default.svc.cluster.local` service and its destination rules as seen from the some a pod of the `sleep` app, use the following commands:
```

## Verify mutual TLS configuration

> For this task, you may want to get the pod ID of the client app (`sleep`) with the following command:

Reviewer: Will readers need that information later? You are even storing the value in and environment variable so my guess is yes. 
What do readers need to change in the command? (my guess is nothing but I have to ask)
I suggest you let readers know what the command is doing and why they need to run it:
```suggestion
This task uses the pod ID of the `sleep` client app. Store it in the `SLEEP_POD` environment variable with the following command:
```

Author: I move this command to the related block to keep them closed.

{{< text bash >}}
$ SLEEP_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
{{< /text >}}

> You can use the `istioctl` tool to check the effective mutual TLS settings. The command requires the client pod as the destination rule depends on the client namespace.

Reviewer: Should readers do these things now? Later? What command are you talking about, the one on line 84?

Author: Yes.

Reviewer: ```suggestion
Use the `istioctl` tool to check if the mutual TLS settings are in effect. The `istioctl` command needs the client's pod because the destination rule depends on the client's namespace.
```

> You can use the `istioctl` tool to check the effective mutual TLS settings. The command requires the client pod as depend on the client namespace, the destination rule for it can be different.

Reviewer 2: Grammar

Author: Done

You can also provide the destination service to filter the status to that service only.

> For example, to identify the authentication policy for service `httpbin.default.svc.cluster.local` and destination rules to it, as seen from pod `${SLEEP_POD}`, use the following command:

Reviewer: Be consistent with the rest of the docs: 
```suggestion
To identify the authentication policy for the `httpbin.default.svc.cluster.local` service and its destination rules as seen from the `${SLEEP_POD}` pod, use the following command:
```

Author: Done

## Verify mutual TLS configuration

Use the `istioctl` tool to check if the mutual TLS settings are in effect. The `istioctl` command needs the client's pod because the destination rule depends on the client's namespace.
You can also provide the destination service to filter the status to that service only.

> To identify the authentication policy for the `httpbin.default.svc.cluster.local` service and its destination rules as seen from the some a pod of the `sleep` app, use the following commands:

Reviewer: This text is still difficult to understand.
```suggestion
The following commands identify the authentication policy for the `httpbin.default.svc.cluster.local` service and identify the destination rules for the service as seen from the same pod of the `sleep` app:
```
